canonical/ubuntu.com
# Update meganav.yaml

## Done

- Updates meganav.yaml based on the [spreadsheet](https://docs.google.com/spreadsheets/d/1XDiSTWRzM9bg1G1cRSMWocD9cgl-fRPmgjwbZ1uZOAI/edit#gid=1407817173)
Tabs updates are as follows:
- [x] products
- [ ] use cases(TBC)
- [x] support 
- [x] community
- [x] get ubuntu (waiting for it to be finalised)




## QA
**note that this is pre rebase, as we are waiting on that to be reviewed**
- Compare meganav.yaml against the [spreadsheet](https://docs.google.com/spreadsheets/d/1XDiSTWRzM9bg1G1cRSMWocD9cgl-fRPmgjwbZ1uZOAI/edit#gid=1407817173)
- Check [the demo](https://ubuntu-com-12967.demos.haus/) is rendering properly

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4298


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
